unblob/handlers/archive/tar.py

```diff
@@ -214,10 +214,10 @@ class TarUnixHandler(_TarHandler):
             r""
             #  (pattern would be too big)   char name[100]
             + _padded_field(r"[0-7]", 8)  # char mode[8]
-            + _padded_field(r"[0-9]", 8)  # char uid[8]
-            + _padded_field(r"[0-9]", 8)  # char gid[8]
-            + _padded_field(r"[0-9]", 12)  # char size[12]
-            + _padded_field(r"[0-9]", 12)  # char mtime[12]
+            + _padded_field(r"[0-7]", 8)  # char uid[8]
+            + _padded_field(r"[0-7]", 8)  # char gid[8]
+            + _padded_field(r"[0-7]", 12)  # char size[12]
+            + _padded_field(r"[0-7]", 12)  # char mtime[12]
```

Comment:
Nitpick: I don't think keeping this in a sepaarate commit adds any value.

Reply:
agreed, this commit was kept separate for review, it is meant to be either dropped or squashed.